### PR TITLE
Feature/implement player adapter and router

### LINKED ISSUE #42: Implement Player Adapter and Router Stubs
Implement stubs for title classes for gameManager to use

### PR DESCRIPTION
**Pull request by wujuu**
Resolves #42

### CODE REVIEW

File: server/src/main/kotlin/core/adapters/PlayerRouter.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package core.adapters
+
+import core.bettinground.ActionValidation
+import core.bettinground.BettingAction
+import core.bettinground.Fold
+import core.gameflow.handstate.HandState
+
+class PlayerRouter {
+    fun getAction(playerId: Int): BettingAction = Fold
+    fun sendPrivateUpdate(playerId: Int, actionValidation: ActionValidation) {}
```
Comment (meszszi) [inline server/src/main/kotlin/core/adapters/PlayerRouter.kt]: I would rename it to excplicit 'sendActionValidation'

File: server/src/main/kotlin/core/adapters/PlayerRouter.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    fun sendPrivateUpdate(playerId: Int, handState: HandState) {}
```
Comment (meszszi) [inline server/src/main/kotlin/core/adapters/PlayerRouter.kt]: Why would we ever want to send an update of the hand state secretly to only one player?

File: server/src/main/kotlin/core/adapters/PlayerRouter.kt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    fun broadcastPlayerAction(playerId: Int, action: BettingAction) {}
+    fun broadcast(handState: HandState) {}
```
Comment (meszszi) [inline server/src/main/kotlin/core/adapters/PlayerRouter.kt]: 'broadcastHandState' would be more readably imo

Comment (CrunchyNomNom) [reply server/src/main/kotlin/core/adapters/PlayerRouter.kt]: legible*

Comment (meszszi) [reply server/src/main/kotlin/core/adapters/PlayerRouter.kt]: readable*

**Pull request merged**